tajo-cli/src/main/java/org/apache/tajo/cli/tsql/commands/ConnectDatabaseCommand.java

```diff
@@ -63,7 +63,7 @@ public void invoke(String[] cmd) throws Exception {
 
   @Override
   public String getUsage() {
-    return "";
+    return "[database_name]";
```

Comment:
I write ```database_name``` refer to ```DescTableCommand::getUsage()```. In ```DescTableCommand::getUsage()```, return ```[table_name]``` in spite of ```[TBNAME]``` with ```\help``` . You mean that editting other ```getUsage()``` is needed?